deployments/properties/azure-kubernetes-service-kompose.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy to AKS with Kompose",
+    "description": "Deploy an application to an Azure Kubernetes Service Cluster using Azure Credentials and Kompose",
+    "creator": "Microsoft Azure",
+    "iconName": "azure",
+    "categories": ["Deployment", "Kompose", "Kubernetes", "Dockerfile"]
```

Comment:
Currently, we don't have detection logic for `Kompose` ,so this workflow will not be recommended.  Can you open an issue in this [repo](https://github.com/github-starter-workflows/repo-analysis-partner) and specify how it should get detected.